README.md

```diff
@@ -60,30 +60,54 @@ benefiting the entire application stack.
 
 ## Build and Verify
 
+### Prerequisites
+
 `Mado` is built with a minimalist design in mind. However, its verification
 relies on certain third-party packages for full functionality and access to all
-its features. To ensure proper operation, the development environment should
-have the [SDL2 library](https://www.libsdl.org/), [libjpeg](https://www.ijg.org/), and [libpng](https://github.com/pnggroup/libpng) installed.
+its features. We encourage the development environment to be installed with all optional
+packages, including [libjpeg](https://www.ijg.org/), [libpng](https://github.com/pnggroup/libpng),
+and the [SDL2 library](https://www.libsdl.org/).
 * macOS: `brew install sdl2 jpeg libpng`
 * Ubuntu Linux / Debian: `sudo apt install libsdl2-dev libjpeg-dev libpng-dev`
 
-Configure via [Kconfiglib](https://pypi.org/project/kconfiglib/)
+### Configure the demo program
```

Comment:
Refine the subject. Demo programs are not the major targets. Instead, users are allowed to build a static library only.